### ISSUE TITLE
Following "Linux installation from sources", missing pip dependency for colcon build

### ISSUE THREAD

**Issue by gstangel** [eProsima/Fast-DDS#4933]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

TLDR: Need to add `pip3 install -U colcon-gradle` to install instructions, 
### Expected behavior

Expected Behavior - able to follow install steps without errors

mkdir ~/Fast-DDS
cd ~/Fast-DDS
wget https://raw.githubusercontent.com/eProsima/Fast-DDS/master/fastdds.repos
mkdir src
vcs import src < fastdds.repos
colcon build
* Colcon builds without warnings *

### Current behavior

Colcon throws error about missing package

mkdir ~/Fast-DDS
cd ~/Fast-DDS
wget https://raw.githubusercontent.com/eProsima/Fast-DDS/master/fastdds.repos
mkdir src
vcs import src < fastdds.repos
colcon build
[1.885s] WARNING:colcon.colcon_core.verb:No task extension to 'build' a 'gradle' package

### Steps to reproduce

Follow steps in https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_linux.html

### Fast DDS version/commit

v2.14.1

### Platform/Architecture

Other. Please specify in Additional context section.

### Transport layer

Default configuration, UDPv4 & SHM

### Additional context

The fix was to run `pip3 install -U colcon-gradle`

**Comment by Mario-DL** [comment]
Hi @gstangel,
Thanks for using `Fast DDS` ! While it is true that `fastddsgen` is also listed in the `fastrtps.repos`, hence discovered by `colcon` during compilation, it is meant to be an independent tool (in fact, we may consider removing it from the `.repos` file). Anyways, we will internally consider your suggestion.

**Comment by Mario-DL** [comment]
I am feeling free to close this issue as the above PR fixes the issue

**Comment by gstangel** [comment]
> I am feeling free to close this issue as the above PR fixes the issue

Thank you!
